cluster_autoscaler.tf

```
@@ -109,11 +116,12 @@ locals {
     "1.20" = "v1.20.1",
     "1.21" = "v1.21.1",
     "1.22" = "v1.22.1",
+    "1.23" = "v1.23.0"
```

```suggestion
    "1.23" = "v1.23.1"
```

The .1 is released. Should we use it?

---

1.24.0 and 1.25.0 is also released. Should we add them to the map?